docs/deployment/manifest-format.md

```diff
@@ -398,6 +556,104 @@ Example manifest:
 }
 ```
 
+#### MongoDB resource types
+
+Example code:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddMongoDB("mongodb1");
+
+builder.AddMongoDBContainer("mongodb2")
+       .AddDatabase("shipping");
+```
+
+Example manifest:
+
+```json
+{
+  "resources": {
+    "mongodb1": {
+      "type": "mongodb.server.v0"
+    },
+    "mongodb2": {
+      "type": "container.v0",
+      "image": "mongo:latest",
+      "bindings": {
+        "tcp": {
+          "scheme": "tcp",
+          "protocol": "tcp",
+          "transport": "tcp",
+          "containerPort": 27017
+        }
+      },
+      "connectionString": "{mongodb2.bindings.tcp.host}:{mongodb2.bindings.tcp.port}"
+    },
+    "shipping": {
+      "type": "mongodb.database.v1",
+      "parent": "mongodb2"
+    }
+  }
+}
+```
+
+#### MySQL resource types
+
+Example code:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddMySql("mysql1");
+
+builder.AddMySql("mysql2")
+       .AddDatabase("shipping");
```

Comment:
Couldn't you chain this?

```suggestion
builder.AddMySql("mysql1")
       .AddMySql("mysql2")
       .AddDatabase("shipping");
```

Reply:
No, since the `AddMySql` cannot be chained on the return type from itself.